validator/sawtooth_validator/execution/processor_handlers.py

```diff
@@ -26,6 +26,11 @@
 LOGGER = logging.getLogger(__name__)
 
 DEFAULT_MAX_OCCUPANCY = 10
+# This is the version used by SDK to match if validator supports feature it
+# requested during registration. It should only be incremented when there are
+# changes in TpRegisterRequest. Remember to sync this information in SDK if
+# changed.
+SDK_PROTOCOL_VERSION = 1
```

Comment:
This will have to be a min/max at some point. Right now, this is treated as a maximum, which would make this MAX_SDK_PROTOCOL_VERSION.

Reply:
Yes, I think we will want MIN eventually. Even without it though, this is really the max protocol version so the name makes sense either way.

Reply:
Ok